conans/test/integration/remote/multi_remote_test.py

```diff
@@ -137,7 +137,7 @@ def test_conan_install_update(self):
                       client.out)
 
         client.run("install --reference=hello0/0.0@lasote/stable --update -r default")
-        self.assertIn("hello0/0.0@lasote/stable#4963e851819c02f9578b9d6bcecd01aa - Cache",
+        self.assertIn("hello0/0.0@lasote/stable#4963e851819c02f9578b9d6bcecd01aa - Newer",
```

Comment:
This test was returning cache because it was searching in all remotes, including local, but that was a bug, if you add the argument -r default it should only check in default, not in all remotes.

Reply:
Yes, I observed that bug lately